package.json

```
@@ -176,4 +175,4 @@
       "moduleResolution": "node"
     }
   }
-}
+}
```

Strange that this newline was removed but no others, was this intentional?

---

```suggestion
}

```

Can github handle a whitespace suggestion.. let's find out 😅